ARPG/Assets/Scripts/PlayerStats.cs

```diff
@@ -0,0 +1,86 @@
+using System;
+using UnityEngine;
+
+public class PlayerStats : MonoBehaviour, IDamageable
+{
+    //hp variables
+    public float baseMaxHealth = 100f;
+    public float maxHealthModPerc = 1f;
+    public float MaxHealth => baseMaxHealth * maxHealthModPerc;
+
+    
+    public float currentHealth; //for clamp functionality
+    public float CurrentHealth
+        { get => currentHealth; private set => currentHealth = Math.Clamp(value, 0, MaxHealth); } 
+
+    //mana variables
+    public float baseMana = 100f;
+    public float maxManaModPerc = 1f;
+    public float MaxMana => baseMana * maxManaModPerc;
+
+    public float currentMana; // for clamp functionality
```

Comment:
Same thing as with the currentHealth.

Reply:
see currentHealth

Reply:
resolved during meeting to satisfaction of... probably everyone, i'd hope!